recipes/gatk/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+BINARY_HOME=$PREFIX/bin
+PACKAGE_HOME=$PREFIX/opt/$PKG_NAME-$PKG_VERSION
+
+mkdir -p $PREFIX/bin
+mkdir -p $PACKAGE_HOME
+
+cp $RECIPE_DIR/gatk.sh $PACKAGE_HOME/gatk.sh
+
+SOURCE_FILE=$RECIPE_DIR/gatk-register.sh
+DEST_FILE=$PACKAGE_HOME/gatk-register.sh
+
+
+echo "#!/bin/bash" > $DEST_FILE
+echo "PKG_NAME=$PKG_NAME" >> $DEST_FILE
```

Comment:
Is there a better way to get the conda package variables into a non-build script?